src/gc/env/gcenv.base.h

```diff
@@ -175,8 +181,130 @@ typedef DWORD (WINAPI *PTHREAD_START_ROUTINE)(void* lpThreadParameter);
  #endif
 #else // _MSC_VER
 
+// Only clang defines __has_builtin, so we first test for a GCC define
+// before using __has_builtin.
+
+#if defined(__i386__) || defined(__x86_64__)
```

Comment:
This section defines compiler intrinsic wrappers and other helpers used in the GC (`ALIGN_UP` and `ALIGN_DOWN`) that are normally defined by the EE.